app/src/main/res/layout/main_nav_header.xml

```diff
@@ -55,12 +55,13 @@
                     android:background="@drawable/circle_placeholder" />
 
                 <TextView
+                    android:id="@+id/getName"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/spacing_normal"
                     android:text='@{ user.firstName + " " + user.lastName }'
-                    android:textColor="@android:color/white"
                     android:textAppearance="@style/TextAppearance.AppCompat.Small.Inverse"
-                    android:layout_marginTop="@dimen/spacing_normal"/>
+                    android:textColor="@android:color/white" />
```

Comment:
This is virtually same as previous one.

Reply:
The rotation is not disabled. Please check again
Here is the screenshot of the activity when the device is rotated.

![screenshot_20180225-101436](https://user-images.githubusercontent.com/23053724/36638230-2ed34824-1a15-11e8-960c-69da588a2fc9.png)

Reply:
It forces the fragments and activities to not get destroyed on rotation. The default rotation behaviour is disabled. This is a hack and not a fix